bin/gobuild.sh

```diff
@@ -36,7 +36,7 @@ GOBINARY=${GOBINARY:-go}
 GOPKG="$GOPATH/pkg"
 BUILDINFO=${BUILDINFO:-""}
 STATIC=${STATIC:-1}
-LDFLAGS="-extldflags -static"
+LDFLAGS="-extldflags -static -s -w"
```

Comment:
This will use the options for all the binaries, not just _istioctl_.  This might be a good thing ... smaller container images for control plane ... but the backtraces may change slightly.  If you only want to change _istioctl_ you'll need to change the _Makefile_ to set LDFLAGS explicitly when calling _gobuild.sh_ for all _istioctl_ builds.